### PR TITLE
updated workflows - lts

### PR DESCRIPTION
**Pull request by ATorrise**
Action | Description
-- | --									
#6 | [update all workflows to remove deprecation warnings](https://github.com/zowe/zowe-cli-web-help-generator/pull/35/files), [octorelease updates](https://github.com/zowe/zowe-cli/pull/1712)										
#7 | remove Node 14 from workflow matrix

### CODE REVIEW

**Review by t1m0thyj — CHANGES_REQUESTED** [review]
There are still some warnings to resolve for the Imperative CI workflow: https://github.com/zowe/imperative/actions/runs/4895746593
* `codecov/codecov-action`: v2 -> v3
* `t1m0thyj/unlock-keyring`: master -> v1

File: .github/workflows/audit.yml
Comment (zFernand0) [inline .github/workflows/audit.yml:1]: |If it's ok, can we add the `zowe-v?-lts` branch to the following workflows:
- audit
- changelog
- linter (just needs to change the `v1` with `v?` 😋 

Here is an example
```yaml
on:
  pull_request:
    branches:
      - master
      - zowe-v?-lts
```

**Review by zFernand0 — CHANGES_REQUESTED** [review]
The changes LGTM.
I left one small comment though 😋

**Review by t1m0thyj — APPROVED** [review]

File: .github/workflows/changelog.yml
Comment (zFernand0) [inline .github/workflows/changelog.yml:1]: I believe I missed this one
Can we update the `awharn/check_changelog_action@v0.0.2` to `v1`?

**Review by zFernand0 — CHANGES_REQUESTED** [review]

**Review by zFernand0 — APPROVED** [review]

**Pull request merged**